perf.groovy

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+// Import the utility functionality.
+
+import jobs.generation.ArchivalSettings;
+import jobs.generation.Utilities;
+
+def project = GithubProject
+def branch = GithubBranchName
+
+def static getBuildJobName(def configuration, def os) {
+    return configuration.toLowerCase() + '_' + os.toLowerCase()
+}
+
+// Setup perflab tests runs
+[true, false].each { isPR ->
+    ['Windows_NT'].each { os ->
+        ['x64', 'x86'].each { arch ->
+            def architecture = arch
+            def jobName = "perf_perflab_${os}_${arch}"
```

Comment:
Probably want to update the name of the job to better reflect the SDK. Something like SDK_Perf_${os}_${arch}

Reply:
Done...